### PR TITLE
MIP53

### PR DESCRIPTION
**Pull request by Planet-X-github**
MIP#: 53
Title: The MKR Compensation Guidelines
Author(s): @Planet_X
Contributors: @Derek, @LongForWisdom, @Aes
Tags: General
Type: General
Status: RFC
Date Proposed: 2021-11-05
Date Ratified: TBA
Dependencies:
Replaces:

### CODE REVIEW

**Comment by CPSTL** [comment]
@Planet-X-github 
This will need some formatting cleanup - namely markdown edits. While we update the proposal, can you please post the MIP on the forums as well: https://forum.makerdao.com/c/mips/conception/20 ?

File: mip53.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+**MIP53c3: Recommended best practice**
+
+Describes recommended practices by the Core Units.
+
+**MIP53c4: The MKR Compensation Guidelines towards 2025**
+
+Describes the planned evolution of the guidelines.
+
+**MIP53c5: Amending the MKR Compensation Guidelines**
```
Comment (Davidutro) [inline mip53.md:45]: missing a description for line49 @Planet-X-github

**Comment by Davidutro** [comment]
@Planet-X-github 

- Please add a description for line 49

Apart from that, I took care of everything afaict. Will merge once that description is pushed.

**Comment by Planet-X-github** [comment]
@Davidutro  did I manage to add the description? I am total github newbie...

**Comment by Davidutro** [comment]
That's perfect

**Pull request merged**